build_tools/python/benchmark_suites/iree/cuda_benchmarks.py

```diff
@@ -46,14 +46,14 @@ def _generate_configs(
       compile_config: iree_definitions.CompileConfig,
       execution_config: iree_definitions.
       ModuleExecutionConfig = module_execution_configs.CUDA_CONFIG,
-      run_tags: Sequence[str] = [],
+      tags: Sequence[str] = (),
   ) -> Tuple[List[iree_definitions.ModuleGenerationConfig],
              List[iree_definitions.E2EModelRunConfig]]:
     gen_configs = [
         iree_definitions.ModuleGenerationConfig.build(
             compile_config=compile_config,
-            imported_model=iree_definitions.ImportedModel.from_model(model))
-        for model in models
+            imported_model=iree_definitions.ImportedModel.from_model(model),
+            tags=tags) for model in models
```

Comment:
Instead of using tags, I think we should have a field "presets" on module generation and run configs to indicate which benchmark preset they belong to. I'm planning to do some overhaul changes later to refactor these.

Here I'm trying to implement with what we have now to unblock https://github.com/openxla/iree/pull/13581

Reply:
Hmmm but that means each model only belongs to exactly one preset